lib/tasks/setup.rake

```diff
@@ -3,80 +3,19 @@ require 'zip'
 namespace :setup do
   desc "Download and/or unpack some dependencies for vita-min"
 
-  EFILE_SCHEMAS_FILENAMES = [
-    ["efile1040x_2020v5.1.zip", "irs"],
-    ["efile1040x_2021v5.2.zip", "irs"],
-    ["efile1040x_2022v5.3.zip", "irs"],
-    ["efile1040x_2023v5.0.zip", "irs"],
-    ["NYSIndividual2023V4.0.zip", "us_states"],
-    ["AZIndividual2023v1.0.zip", "us_states"],
-  ].freeze
+  VENDOR = "vendor"
```

Comment:
Could you change this to VENDOR_DIR or VENDOR_PATH?

Reply:
Done